src/main/cpp/blaze.cc

```diff
@@ -359,13 +359,12 @@ static vector<string> GetServerExeArgs(const blaze_util::Path &jvm_path,
   startup_options.AddJVMArgumentPrefix(jvm_path.GetParent().GetParent(),
                                        &result);
 
-  // TODO(b/109998449): only assume JDK >= 9 for embedded JDKs
-  if (!startup_options.GetEmbeddedJavabase().IsEmpty()) {
-    // quiet warnings from com.google.protobuf.UnsafeUtil,
-    // see: https://github.com/google/protobuf/issues/3781
-    result.push_back("--add-opens=java.base/java.nio=ALL-UNNAMED");
-    result.push_back("--add-opens=java.base/java.lang=ALL-UNNAMED");
-  }
+  // quiet warnings from com.google.protobuf.UnsafeUtil,
+  // see: https://github.com/google/protobuf/issues/3781
```

Comment:
This shouldn't actually be necessary anymore. Are these flags necessary for protobuf, or something else that's doing reflection? Or maybe bazel is on an old version of protobuf? https://github.com/protocolbuffers/protobuf/commit/624a40a387e1b2da4c6bbde44ad3d0a5efb9879b#diff-b05e35be325231f2f3d063254c4171b1e76c92a2fb4496d67dc061e22e71fcf4L401

Reply:
I see from the linked bugs that it's needed for `com.google.devtools.build.lib.unsafe.StringUnsafe`. Can you update the comment?

Reply:
I see from the linked bugs it's needed for `com.google.devtools.build.lib.unsafe.StringUnsafe`, can you update the comment?

Reply:
I updated the comment and removed the `java.nio` opens as it seems unused.